dev/messaging.env.default

```diff
@@ -1,11 +1,6 @@
 # Example docker-compose environment file
-# Copy to pubclient.env and modify as necessary
+# Copy to messaging.env and modify as necessary
```

Comment:
My YAGNI sense is tingling, but it's been very sensitive around these *-environments, so feel free to ignore :)